doc/content/integrations/cloud-integrations/datacake/datacake-setup/_index.md

```diff
@@ -12,14 +12,27 @@ First, create a **Workspace** on Datacake by navigating to the **Create Workspac
 
 {{< figure src="datacake-workspace.png" alt="Add workspace on Datacake" >}}
 
-On the left hand menu, click **Devices**. To add a new device, click on **Add Device** button on the right. 
+On the left hand menu, click **Devices**. To add a new device, click the **Add Device** button on the right. 
 
-In the **Add Device** pop-up menu, choose **LoRaWAN** &#8594; **Generic LoRa Device** &#8594; **The Things Industries** adapter.
+In the **Add Device** pop-up menu, choose **LoRaWAN** &#8594; **Generic LoRa Device**. When asked **Which device are you missing?**, just click **Skip** and then select **The Things Industries** adapter.
 
 {{< figure src="datacake-tti-adapter.png" alt="Datacake TTI adapter" >}}
 
-Fill in **Name** and **DevEUI** for your device, then click on **Add Device** to finish.
+After selecting your subscription plan, fill in **Name** and **DevEUI** for your device, then click on **Add Device** to finish.
 
 Once the device is created, you can click on it in the **Devices** menu to enter its settings.
 
-In the **Configuration** tab, you can find **Payload Decoder**, which is intended for [decoding the payload](https://docs.datacake.de/lorawan/payload-decoders) received from {{% tts %}} after deploying the integration. In order to store the payload values you need to [create a field](https://docs.datacake.de/device/database/fields) according to the payload type. 
+In the **Configuration** tab, you can find **LORaWAN** section, where you can configure **Network** settings, choose to **Authenticate Webhook** or define **Payload Decoder**. 
+
+Under **Network**, choose **The Things Industries** from the drop-down menu.
```

Comment:
There is a screenshot at the end of this file which covers all of these steps (it seems like a lot of steps because every field from LoRaWAN section is described in detail). I could only add a screenshot for creating an API key on TTS or the one from Datacake that shows the Name and DevEUI fields, but I actually think that would be irrelevant. What do you think?